### ISSUE TITLE
JSON.Stringify object output?

### ISSUE THREAD

**Issue by evilC** [DVLP/run-func#9]
Could you possibly add an option to check if the return value is an object, and if so, `JSON.stringify` it? (Ideally have a way to set stringify options too - eg allow pretty printing)
As it stands, if the return value is an object, it only shallow prints it, so I just get a bunch of `[Object]`

**Comment by whaaaley** [comment]
I agree. Though I would call it `--inspect` and it would use node's inspect util.
